src/lib.rs

```diff
@@ -830,8 +830,8 @@ impl<T> Slab<T> {
     /// assert_eq!(slab[key2], 1);
     /// ```
     pub unsafe fn get2_unchecked_mut(&mut self, key1: usize, key2: usize) -> (&mut T, &mut T) {
-        let ptr1 = self.entries.get_unchecked_mut(key1) as *mut Entry<T>;
-        let ptr2 = self.entries.get_unchecked_mut(key2) as *mut Entry<T>;
+        let ptr1 = self.entries.as_mut_ptr().add(key1);
+        let ptr2 = self.entries.as_mut_ptr().add(key2);
```

Comment:
Please call `as_mut_ptr()` once and reuse it. 
```suggestion
        let ptr = self.entries.as_mut_ptr();
        let ptr1 = ptr.add(key1);
        let ptr2 = ptr.add(key2);
```